app/helpers/bgeigie_imports_helper.rb

```diff
@@ -18,7 +18,7 @@ def bgeigie_nav_li(status) # rubocop:disable Metrics/AbcSize
                params[:by_status] == status.to_s
              end
     content_tag(:li, class: ('active' if active)) do
-      p = params.except(:action, :controller).merge(by_status: (status unless status == :all))
+      p = params.permit!.except(:action, :controller).merge(by_status: (status unless status == :all))
```

Comment:
I believe that `permit!` is called to get `Hash`-like object. I think that `#to_h` or `#to_unsafe_h` would be better to get `Hash`-like object.

Reply:
`to_h` won't work because of `unable to convert unpermitted parameters to hash`. I'll try to use `to_unsafe_h`

Reply:
@eitoball Ready a6ff470